fuse.py

```diff
@@ -376,18 +377,27 @@ class fuse_operations(Structure):
         ('flag_reserved', c_uint, 29),
     ]
 
+# When importing "fuse", you can check this with "hasattr(fuse, 'NANOSECOND_INT_AVAILABLE')"
+NANOSECOND_INT_AVAILABLE = True
```

Comment:
I am on the fence about the explicit module level feature flag for `NANOSECOND_INT_AVAILABLE`. I like that it makes the code introspective and self-documenting. I do not like that it further clutters the global module namespace and adds another thing to maintain (i.e. making sure the flag doesn't get out of sync with the true state of the code itself). I see two possible solutions:

1. Simply use the version number to indicate the change. With this change, the minor version number would simply be [increased to indicate a backwards compatible feature addition](https://semver.org/). Anyone then needing this particular feature could/should indicate they need the appropriate version (or greater) of the package in their `requirements.txt` file.
2. Add a module level `__features__` dictionary to hold all potential features flags. This avoids global namespace clutter, and also helps with maintenance a little (since its name should be a clue to future maintainers).

I would also suggest the comment above use `getattr(fuse, 'NANOSECOND_INT_AVAILABLE', None)` instead of `hasattr(fuse, 'NANOSECOND_INT_AVAILABLE')`. The `hasattr` approach runs the risk that the feature flag is present, but set to `False`, creating a subtle and hard to catch bug.

Reply:
I like solution 2, it's actually a really good idea. It makes sense with more potential future features in mind. Going by version numbers ... no. Working with ``getattr`` also makes sense. I had similar concerns but ended up hacking this "attribute without caring about its value" thing ... Ok, I will adjust this PR. Thanks for the review.

Reply:
Really, the version number is going to happen either way to make sure people can lock off on a version and know what they are getting. So really, the choices were 1. Don't do it (since versions are there either way) or 2. Do it a slightly different way. I would also suggest adding `NANOSECOND_FLOAT_AVAILABLE` with a value of `True`. When it is removed (sometime in the future), the `FLOAT` flag will be set to `False`.